actions/check_if_member_MOD.js

```diff
@@ -136,6 +136,8 @@ module.exports = {
     const { Files } = this.getDBM();
     const { msg } = cache;
 
+    if (!member) return console.error('You need to provide a member of some sort.');
```

Comment:
I believe we should still continue the action sequence. Perhaps a more informative error message would help too (at least show the action name)?